Cargo.toml

```diff
@@ -14,6 +14,7 @@ edition = "2018"
 default = []
 vhost-vsock = []
 vhost-kern = []
+vhost-vdpa = []
```

Comment:
Should vhost-vdpa features select vhost-kern?
vhost-vdpa = ["vhost-kern"]

Reply:
Yep! I'll fix!